### PR TITLE
Paint tool on startup fixes

### PR DESCRIPTION
**Pull request by PaulHax**
Small paint tool fixes:
- If session.volview.zip is loaded with the paint tool initially selected, clicking on the image did not paint.
- If paint tool is activated when mouse is within a view, the stencil shape would not be correct.

### CODE REVIEW

File: src/store/tools/paint.ts [hunk 1]
```diff
@@ -80,18 +80,6 @@ export const usePaintToolStore = defineStore('paint', () => {
...
-        throw new Error('Cannot set active segment without a labelmap');
-
-      const segmentGroupStore = useSegmentGroupStore();
-      const { segments } =
-        segmentGroupStore.metadataByID[activeSegmentGroupID.value];
-
-      if (!(segValue in segments.byValue))
-        throw new Error('Segment is not available for the active labelmap');
-    }
-
```
Comment (floryst) [inline src/store/tools/paint.ts:94]: Can you explain how this block of code affected the paint tool not loading properly through a session file?

**Comment by floryst** [comment]
Overall LGTM! My quick functional test didn't reveal any issues. Just one clarifying question.

File: src/store/tools/paint.ts (hunk 1 above)
Comment (PaulHax) [reply src/store/tools/paint.ts:94]: Yes.  `usePaintStore.deserialize` calls `setActiveSegment` before `activeSegmentGroupID` is set.  Later, activeSegmentGroupID will be set by `useToolStore.setCurrentTool` -> `usePaintStore.activateTool` -> `setActiveLabelmapFromImage`

Comment (floryst) [reply src/store/tools/paint.ts:94]: I think then that we can solve this by calling `setActiveLabelmap(segmentGroupIDMap[paint.activeSegmentGroupID])` before `setActiveSegment` in `deserialize`. I think keeping the code in `setActiveSegment` is good to ensure we are getting a valid segment value. Thoughts?

Comment (PaulHax) [reply src/store/tools/paint.ts]: Would be nice to keep the checks.   I rebased a fix.  Turns out segmentIDMap was not getting passed down to `paint.deserialize`.  Also, can't `setActiveSegment` without a segmentGroup, so only calling `setActiveSegment` if one is deserialized.

**Pull request merged**